expr/func_movingaverage.go

```diff
@@ -1,41 +0,0 @@
-package expr
-
-import (
-	"github.com/grafana/metrictank/api/models"
-)
-
-type FuncMovingAverage struct {
-	window int64
-	in     GraphiteFunc
-}
-
-func NewMovingAverage() GraphiteFunc {
-	// TODO: allows both int and string as window. how to handle properly?
-	return &FuncMovingAverage{}
-}
-
-// note if input is 1 series, then output is too. not sure how to communicate that
-func (s *FuncMovingAverage) Signature() ([]Arg, []Arg) {
-	return []Arg{
-		ArgSeriesList{val: &s.in},
-		// this could be an int OR a string.
-		// we need to figure out the interval of the data we will consume
-		// and request from -= interval * points
-		// interestingly the from adjustment might mean the archive TTL is no longer sufficient and push the request into a different rollup archive, which we should probably
-		// account for. let's solve all of this later.
```

Comment:
this problem seems to have been solved thanks to the context from adjustment.
e.g. with 6h retention, 3h of recent data + 3h window fetches raw, but 4h window goes to the first rollup